### ISSUE TITLE
Add existing question and extend optionset for "data-usage/scenarios-usage/infrastructure"

### ISSUE THREAD

**Issue by smohex** [FDM-UARuhr/rdmo-catalog-uaruhr#58]
**Describe the request**  
Information about the storage (slow storage , high performance storage ) is still missing but important for the storage colleagues

**Propose what exactly should be changed**  
New question can may be integrated into the existing question set ua_ruhr_v1.1.0-beta.1/data-usage/scenarios-usage

**Proposed setting or content**   
*question text ger*
~~"Wie viele Daten werden gleichzeitig in den Speicher geladen. Wie häufig müssen die Daten gelesen und/oder geändert werden?"~~
*add option to question text en*  
~~"How many data are loaded into the storage simultaneously. How often must the data be read and/or changed?"~~
* HP Storage is needed (fast read and write of data) [TEXT]

**Additional context**
~~new attribute necessary~~

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [ ]~add question text (en and ger)~
- [x] Extend option set (terms/options/infrastructure_resources)
- [ ] Add question in archive catalog

**Comment by jwindeck** [comment]
similar questions
``data-usage/scenarios-usage/frequency``
``data-usage/scenarios-usage/infrastructure``
#57

**Comment by smohex** [comment]
UDE: the discussion with the storage colleagues was mainly about the part how many data are loaded into the storage simultaneously. Therefore I would agree to add or reformulate the question data-usage/scenarios-usage/frequency. Whereby it may still be necessary to clarify whether the question is to be reduced to a data set or applies to the project in general. The mentioned question is currently not yet available in the archive catalog.
TUDO: Approval and questions in the archive catalog supplement. In any case, the question must be limited to the data set, because within a project there will be different scenarios of data ingestion into the storage

**Comment by jmfrenzel** [comment]
The additional option 
  * "High performance storage is needed (fast read and write of data)"
has to be added to the optionset  'https://rdmorganiser.github.io/terms/options/infrastructure_resources'

I Propose the following steps to implement the changes 
1 .We should do a PR  upstream https://github.com/rdmorganiser/rdmo-catalog to keep the maintenance of this fork low https://github.com/rdmorganiser/rdmo-catalog/issues/74
2. add the question to the archive catalog

**Comment by jmfrenzel** [comment]
https://github.com/rdmorganiser/rdmo-catalog/issues/74

**Comment by ClaraRosalie** [comment]
implemented in upstream rdmorganiser#87
